packages/jest-jasmine2/src/index.js

```diff
@@ -69,7 +69,32 @@ function jasmine2(config, environment, moduleLoader, testPath) {
       moduleLoader.requireModule(null, config.setupTestFrameworkScriptFile);
     }
   });
+
+  const iterableEquality = (left, right) => {
```

Comment:
In most of Jest, when we compare two things we usually call them just `a` and `b`.